tests/cypress/support/commands.js

```diff
@@ -15,15 +15,26 @@ import { pageLoader, isPolicyStatusAvailable, isClusterPolicyStatusAvailable, is
          action_checkNotificationMessage
 } from '../common/views'
 
-Cypress.Commands.add('login', (OPTIONS_HUB_USER, OPTIONS_HUB_PASSWORD, OC_IDP) => {
-  var user = process.env.SELENIUM_USER || OPTIONS_HUB_USER || Cypress.env('OPTIONS_HUB_USER')
-  var password = process.env.SELENIUM_PASSWORD || OPTIONS_HUB_PASSWORD || Cypress.env('OPTIONS_HUB_PASSWORD')
-  var idp = OC_IDP || Cypress.env('OC_IDP')
+Cypress.Commands.add('login', (OPTIONS_HUB_USER='', OPTIONS_HUB_PASSWORD='', OC_IDP='') => {
+  const user = OPTIONS_HUB_USER || process.env.SELENIUM_USER || Cypress.env('OPTIONS_HUB_USER')
+  const password = OPTIONS_HUB_PASSWORD || process.env.SELENIUM_PASSWORD || Cypress.env('OPTIONS_HUB_PASSWORD')
+  const idp = OC_IDP || Cypress.env('OC_IDP')
+  const APIServer = Cypress.env('API_SERVER_URL')
+  cy.clearCookies()
+
+  // handle login using cookie when running against localhost
+  if (Cypress.config().baseUrl.includes('localhost')) {
+    expect(APIServer).to.not.equal(undefined)
+    cy.exec(`oc login --server=${APIServer} -u ${user} -p ${password}`).then(res => cy.log(res.stdout))
```

Comment:
You can do `cy.exec('oc whoami')` to get current username and compare it with given user. Only login again if it's not current user. This will avoid login multiple times

Reply:
Added